src/helpers/Client.js

```diff
@@ -0,0 +1,23 @@
+import Peer from 'peerjs';
+import EventBus from './EventBus';
+
+export default class Client {
+  constructor(hostId) {
+    this.eventBust = EventBus.getInstance();
```

Comment:
Did you mean to call it eventBust (nbd either way, just figured I call out your spelling)

Reply:
nope!